src/models/GroupProvider.ts

```diff
@@ -91,6 +91,28 @@ export class GroupProvider implements TreeDataProvider<CustomTreeItem> {
         this.groups[groupName] = { isActive: false, viewColumn: 1, tabs: [], activeTab: undefined};
         this._onDidChangeTreeData.fire();
     }
+
+    /**
+     * Deletes the selected group
+     * @param groupName 
+     */    
+    deleteGroup(groupName: string): void {
+        if (!this.groups){
+            this.addEmptyGroup('No groups yet');
```

Comment:
instead of adding a group just to be deleted would it make more sense to move this check to the `deleteGroup` method in `utils.ts`?
It's kinda weird to make a group just to delete it, what do you think?